SQL/0000-00-00-schema.sql

```diff
@@ -724,6 +732,38 @@ INSERT INTO mri_protocol (Center_name,Scan_type,TR_range,TE_range,time_range) VA
   ('ZZZZ',44,'2000-2500','2-5',NULL),
   ('ZZZZ',45,'3000-9000','100-550',NULL);
 
+CREATE TABLE `mri_protocol_group` (
```

Comment:
It's difficult to review this PR because these tables are new and  meaning of them/their intended purposes isn't really defined anywhere.. can you maybe put a comment on each explaining what they're for and what they mean, for lack of a better place to document it?

Reply:
@driusan the tables are all documented in the SQL patch included in this PR. I did not add any comments in `0000-00-00-schema.sql` since no table seems to be documented in there.